src/main/util.ts

```diff
@@ -452,6 +452,26 @@ const cutVideo = async (initialFile: string, finalDir: string, desiredDuration:
     });
 }
 
+/**
+ * Gets string value from the config in a more reliable manner.
+ * @param cfg the config store
+ * @param key the key
+ * @returns the string config
+ */
+const getPathConfigSafe = (cfg: any, key: string) : string => {
+  return cfg.has(key) ? path.join(cfg.get(key), "/") : "";
+}
+
+/**
+ * Gets number value from the config in a more reliable manner.
+ * @param cfg the config store
+ * @param preference the preference
+ * @returns the number config
+ */
+ const getNumberConfigSafe = (cfg: any, preference: string) : number => {
+    return cfg.has(preference) ? parseInt(cfg.get(preference)) : -1;
```

Comment:
Is there something better than -1 we could return here? Realise that for now this is fine, but if we add number config in the future that is allowed to be negative then this breaks. Is there a NaN equivalent in JS?

Reply:
NaN exists. i like it